### PR TITLE
Consolidate Image Adjustments

### PR DESCRIPTION
**Pull request by alexdarling**
## Summary

- Performed a consistency pass on image adjustment sliders, making them all use classes instead of IDs.
- Moved image reset functions into `common-before.js`
- Gives all image adjustment sliders text ranges.
- Removes image reset on image upload for deck fronts, to match multi-image upload behavior.
- Adds `CARD_FORMS` and optional chaining on the maps so that the JS console will stop yelling at me.
- Fixes a mistake in the refactor of `EFFECT_PHASE_FONT_SIZE`.

(this does not change how the reset buttons work on the multi-image vs single-image pages, the former still deletes images on reset while the latter does not).

## Test Plan
On each of the pages:

1. Upload an image (or all images, for the multi-image pages)
2. Fiddle with the zoom, x, and y.
3. Reset. On the deck front pages, verify that the settings reset and the canvas updates to accommodate. On the other pages, verify that the settings reset, the image is removed, and the canvas updates to accommodate.

## Screenshots
image
image
image
image

## Reviewers
@sjzhu 
@Colcoction

### CODE REVIEW

File: environment-deck-front/script.js [hunk 1]
```diff
@@ -597,6 +550,368 @@ function drawCardAttribution() {
   }
 }
 
+
+
+function drawCardEffect() {
```
Comment (sjzhu) [inline environment-deck-front/script.js]: Was this intentional? Looks like a reversion of your previous refactor of card body rendering?

**Review by sjzhu — COMMENTED** [review]
LGTM other than that one comment!

**Review by alexdarling — COMMENTED** [review]
image

merge should be fixed now, grrrr

**Review by sjzhu — APPROVED** [review]
LGTM!

**Pull request merged**